rfcs/text/0000-roadmaps+planning.md

```diff
@@ -0,0 +1,421 @@
+
+## Authors
+
+[Cara Mason](mailto:cara@broadinstitute.org)
+
+[Brian Raymor](mailto:brianraymor@chanzuckerberg.com)
+
+## Contributors
+
+[Trevor Heathorn](mailto:theathor@ucsc.edu)
+
+[Matt Weiden](mailto:mweiden@chanzuckerberg.com)
+
+  The original roadmap process was "prototyped" by the DCP Project Leads.
+
+## Shepherd
+
+  [Brian Raymor](mailto:brianraymor@chanzuckerberg.com)
+
+## Motivation
+
+The DCP enables large-scale impact of centralized data generated from the HCA; however, one of the **challenges** facing the DCP as it matures was identified in this [Update from HCA Science Governance](https://docs.google.com/presentation/d/13hZ7vmr371mN1YFjFyDN-6c0Ul95NwiJcVsjA3nJFVE/edit#slide=id.p3):
+
+> "A lack of reproducible planning process that prioritizes deliverables and ensures confidence that they will meet the needs of HCA"
+
+To deliver on the large-scale impact of centralized data, this RFC defines a transparent, iterative process to plan, align, and execute work across the DCP.
+
+To best serve our users and address the concerns of HCA Science Governance, this process focuses on the needs and priorities of DCP "data as a service", rather than the individual components. This requires a DCP-wide project planning and management process with clear priorities and product increments delivered on a regular cadence.
+
+## Definitions
+
+**Users of the DCP** include: Computational and non-computational consumers of the DCP, scientific data contributors to the DCP, and methods and application developers.
+
+**Project Leads**, **Product Owners**, and **Technical Leads** are roles identified in DCP [charters](https://github.com/HumanCellAtlas/dcp-community/tree/master/charters) with further details available in the [PM](https://github.com/HumanCellAtlas/dcp-community/blob/master/charters/PM/charter.md) and [Architecture](https://github.com/HumanCellAtlas/dcp-community/blob/master/charters/Architecture/charter.md) charters.
+
+**Themes** represent a discrete user or stakeholder need that delivers on the long term vision of the HCA. A Theme motivates a collection of work performed by the DCP.
+
+**Roadmap Objectives** are the concrete elements of an overarching Theme.
+
+**Product Roadmaps** identify and express user needs modeled as high-level Themes that capture user priorities. Roadmaps align the DCP community around a common set of Themes and get scientists excited about future releases.
+
+**Science Roadmaps** communicate a high-level vision of the goals of the single-cell community and a trajectory of progress towards those goals. It identifies both use cases for the constructed Atlas and open scientific questions that must be solved to build it.
+
+**[User Stories](http://agiledictionary.com/277/user-story/)** are requirements "stated as a sentence or two of plain English. A user story is often expressed from the user’s point of view, and describes a unit of desired functionality."
+
```

Comment:
Should we also specify that user stories must specify the type of user being affected?  We've had problems with vague/ over-generalized user stories in the past when this was forgotten (e.g. "as a scientist, i want to...")

Reply:
That sounds like a good suggestion. I did add references to the personas in the roadmap rfc template - *Document how this Objective delivers value to specific [HCA Personas](https://docs.google.com/presentation/d/1bsu8q9CzRXv3Y8c1p8NENi3p8KlxC4scp2tJkHO3Jeo/edit#slide=id.g3cf4e46e8f_0_38).*

 Is this the best reference?

Reply:
That's the right reference.   

I'm ok with using other users beyond the personas as our understanding of our users grows and evolves.  I just want to make sure people are creating good, specific user stories.  Feel free to incorporate that feedback however makes most sense to you.

Reply:
Addressed by https://github.com/HumanCellAtlas/dcp-community/pull/92/commits/8e74b857c69782391aae44b892ffe7ab3ffe413c - @adriennes, please let me know if further detail is required.